templates/events/view_logs.html

```diff
@@ -37,6 +38,7 @@ <h1 class="pg-title">{{ title }}</h1>
                                 </a>
                             </td>
                             <td>{{ event_log.chat_message.content }}</td>
+                            <td>{{ event_log.log }}</td>
```

Comment:
What's the motivation for showing this to the user? They'll just see a stacktrace, which wouldn't be helpful I'd guess? Also, what about sensitive data that might be revealed? :eyes:

Reply:
For errors they would see the result of `str(e)` which is generally just the exception class + message (no stack trace).

For normal events they would see the log message like 'Session ended'.